justfile

```diff
@@ -69,7 +70,9 @@ test-cpu-detection:
 	@# Test crashes on older Qemu versions
 	if qemu-x86_64 -version | grep -Eq 'version [7-9]\.[2-9][0-9]*\.[0-9]+$'; then CGO_ENABLED=0 GOARCH=amd64 go run -exec 'qemu-x86_64 -cpu Haswell' main.go architecture system 2>&1 | grep -q '^x86_64_v3$'; fi
 	@# 32-Bit on x86_64
-	CGO_ENABLED=0 GOARCH=386 go run -exec 'linux32' main.go architecture system | grep -q '^x86_64'
+	CGO_ENABLED=0 GOARCH=386 go run -exec 'linux32' main.go architecture system | grep -q '^i686'
```

Comment:
This change is unrelated und should fail on an x86_64 host. Are you using a 32Bit OS on a x86_64 CPU? If yes, I'll fix this on my end and you may remove that commit.

Reply:
If you rebase your branch on master, these tests run within qemu and might work on non-x86_64 hosts.

Reply:
I have deleted this commit